code/cppRawFtoaFunction/ftoa.h

```diff
@@ -0,0 +1,46 @@
+
+#ifndef FTOA_H
+#define FTOA_H
+
+#include <cstdlib>
+#define MAX_STR_LEN 16
+
+/*
+ * This function returns a char array (string) representation of a floating point value
+ * Parameters: d = double to be converted | precision = number of digits after decimal point
+ */
+char *ftoa(double d, int precision) {
+    char *buffer = (char*)malloc(MAX_STR_LEN * sizeof(char));
+
+    // Add digits before decimal point to string
+    long wholePart = (long)d;
```

Comment:
If you want a better name for the part of a number before the decimal point. It's actual name is "characteristic"

Reply:
It's also reffered to as the "integer-part" in algebra